### PR TITLE
chore: update installation doc for 1.0.0 release

### PR DESCRIPTION
**Pull request by qweeah**
This PR updates the installation doc. Validated Windows and Linux amd64 installation.

### CODE REVIEW

**Review by TerryHowe — APPROVED** [review]
LGTM

File: docs/CLI/index.md [hunk 1]
```diff
@@ -21,37 +21,37 @@ Install ORAS from the latest [release artifacts](https://github.com/oras-project
...
-> Note: If you want to install ORAS on an ARM64-based Linux machine, you can download it from `https://github.com/oras-project/oras/releases/download/v1.0.0-rc.2/oras_1.0.0-rc.2_linux_arm64.tar.gz`.
+> Note: If you want to install ORAS on an ARM64-based Linux machine, you can download it from `https://github.com/oras-project/oras/releases/download/v1.0.0/oras_1.0.0_linux_arm64.tar.gz`.
 
 ### macOS
 
 If you want to install ORAS on a Mac computer with Apple silicon, run the following command:
 
 ```bash
-VERSION="1.0.0-rc.2"
+VERSION="1.0.0"
```
Comment (FeynmanZhou) [inline docs/CLI/index.md:39]: @qweeah I think we need to update this section with Homebrew installation as this has been upgraded in https://github.com/Homebrew/homebrew-core/pull/126141

**Review by FeynmanZhou — CHANGES_REQUESTED** [review]
PTAL at my comment. Thanks

**Review by SteveLasker — APPROVED** [review]
LGTM

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Pull request merged**